<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>972</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item row="0" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="teamBLayout"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="top_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="jug_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mid_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>52</width>
          <height>52</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="adc_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sup_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="teamALayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="team1_win_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1팀 승리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="team2_win_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2팀 승리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>920</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>업로드</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>651</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>920</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="tmp_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Korean Regular</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>870</y>
     <width>651</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="sh45_both_upload_radio_btn">
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4, 5 번 시트 동시입력</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sh5_upload_radio_btn">
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5번 시트 입력(mmr)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sh4_upload_radio_btn">
      <property name="font">
       <font>
        <family>Noto Sans Korean Regular</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4번 시트 입력 (챔피언)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>651</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="result_vlayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
